_tutorials/2016-05-22-getting-started-on-xrv64.md

```diff
@@ -0,0 +1,101 @@
+---
+published: false
+date: "2016-05-22 13:49 -0700"
+title: Getting Started on XRv64
+permalink: "https://xrdocs.github.io/application-hosting/tutorials/xrv64-quickstart"
+author: Lisa Roach
+excerpt: "Getting started with Cisco's XRv64 Vagrant Instance."
+tags: 
+  - vagrant
+  - iosxr
+  - cisco
+  - XRv64
+---
+### Quick Start Using XRv64 Box image
+Quick guide to get up and running with XRv64 and vagrant/Virtualbox.
+ 
+#### Requirements:
+- [Vagrant](https://www.vagrantup.com/) and [Virtualbox](https://www.virtualbox.org/) installed on your laptop
+
+#### Single Node:
+
+	vagrant init xrv64
+  ![vagrant init xrv64](https://xrdocs.github.io/xrdocs-images/assets/tutorial-images/xrv64_vagrant_init.png)
```

Comment:
Unfortunately, these are Rich's screenshots which showcase XRv64 (internal names). Would it be possible for you to try it out on your windows setup and get screenshots from there? And use xrv or iosxrv as the name of your vagrant box.